docs/docs/10.4-test-utils.md

```diff
@@ -12,6 +12,8 @@ next: clone-with-props.html
 var ReactTestUtils = require('react-addons-test-utils');
 ```
 
+>Note: Airbnb has a released a new testing utility called [Enzyme](http://airbnb.io/enzyme/), which makes it easy to assert, manipulate, and traverse your React Components' output.  If you're deciding on a unit testing library, it's worth checking out: [http://airbnb.io/enzyme/](http://airbnb.io/enzyme/)
```

Comment:
1.  I was wondering why the comment was bold :P.  Fixed the boldness.
2.  Eeh, ok, fixed.
3.  Not really.  I think it's pretty clear that Enzyme stands out as a frontrunner (in the context of React).  Did another framework jump out as particularly noteworthy to you?
4.  Fixed.